qemu/tests/cfg/virtio_serial_file_transfer.cfg

```diff
@@ -12,12 +12,20 @@
     host_script = serial_host_send_receive.py
     guest_script = VirtIoChannel_guest_send_receive.py
     Windows:
+        driver_name = vioser
         guest_script_folder = C:\
         clean_cmd = del /f /q
         tmp_dir = %TEMP%
         python_bin = python2.7
     variants:
         - @default:
+            variants:
+                - max_ports:
+                    type = virtio_serial_file_transfer_max_ports
+                    start_vm = no
+                    serials = "serial0 vs0"
+                    serial_type_vs0 = "virtserialport"
```

Comment:
max port number is 31, one is for virtioconsole.
I think vs0 type is better to be virtioconsole

Reply:
one of 31 ports is 'serial0' and its default is isa-serial, I think it is ok, variants format have been updated, please help to review, thanks.

Reply:
> one of 31 ports is 'serial0' and its default is isa-serial, I think it is ok, variants format have been updated, please help to review, thanks.

It's not. isa-serial is different virtio-serial, it's not in 31 ports.
 I think you delete serials = "serial0" is okay

Reply:
> It's not. isa-serial is different virtio-serial, it's not in 31 ports.
> I think you delete serials = "serial0" is okay
Yes, thanks and update.